### PR TITLE
fix online bug: click the top or bottom of filteritem and then auto reselect first item

### PR DESCRIPTION
**Pull request by dyliuti**

### CODE REVIEW

**Comment by ddennedy** [comment]
I do not understand what bug this is trying to fix. “onlien” is not a word. It you meant “online” it does not have meaning here.

File: src/qml/views/filter/AttachedFilters.qml [hunk 1]
```diff
@@ -255,7 +255,12 @@ Rectangle {
                 anchors.fill: attachedFiltersView.contentItem
                 z: 1
                 onClicked: mouse => {
-                    filterClicked(attachedFiltersView.indexAt(mouseX, mouseY));
+                   let index = attachedFiltersView.indexAt(mouseX, mouseY)
+                   if(index < 0) {
```
Comment (ddennedy) [inline src/qml/views/filter/AttachedFilters.qml:259]: Space missing between if and (

Comment (dyliuti) [reply src/qml/views/filter/AttachedFilters.qml]: here is the bug video, and I added space  between if and （

Comment (dyliuti) [reply src/qml/views/filter/AttachedFilters.qml]: > Space missing between if and (

I replied to your email with the bug video

File: src/qml/views/filter/AttachedFilters.qml [hunk 2]
```diff
@@ -255,7 +255,12 @@ Rectangle {
                 anchors.fill: attachedFiltersView.contentItem
                 z: 1
                 onClicked: mouse => {
-                    filterClicked(attachedFiltersView.indexAt(mouseX, mouseY));
+                   let index = attachedFiltersView.indexAt(mouseX, mouseY)
+                   if (index < 0) {
+                       return
+                   }
```
Comment (ddennedy) [inline src/qml/views/filter/AttachedFilters.qml:261]: indentation is wrong

**Review by ddennedy — CHANGES_REQUESTED** [review]

File: src/qml/views/filter/AttachedFilters.qml (hunk 2 above)
Comment (dyliuti) [reply src/qml/views/filter/AttachedFilters.qml]: sorry，I fixed it

Comment (dyliuti) [reply src/qml/views/filter/AttachedFilters.qml]: > indentation is wrong

sorry，I fixed it

**Comment by dyliuti** [comment]
> I did not receive the e-mail with video, but I think I reproduced it. Clicking a row that is a category heading such as Video or Audio changes the selected filter to the first filter.

yes

**Pull request merged**